dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -36,14 +35,25 @@ RUN ln -s -f /usr/bin/gcc-8 /usr/bin/arm-linux-gnueabihf-gcc
 RUN ln -s -f /usr/bin/gcc-8 /usr/bin/aarch64-linux-gnu-gcc
 {% endif %}
 
+# TODO: Remove these lines once restore_neighbors.py is converted to Python 3
+RUN apt-get install -f -y python-dev python-pip
+RUN pip2 install --upgrade 'pip<21'
+RUN apt-get purge -y python-pip
+RUN pip2 install setuptools==40.8.0
+RUN pip2 install wheel==0.35.1
 RUN pip2 install \
          scapy==2.4.2 \
-         setuptools \
-         pyroute2==0.5.3
-RUN pip2 install \
+         pyroute2==0.5.3 \
          netifaces==0.10.7 \
          monotonic==1.5
 
+# Dependencies of restore_neighbors.py
+RUN pip3 install \
```

Comment:
I see since you are adding these packages here,  would you add them to platform/vs/docker-sonic-vs/Dockerfile.j2 as well?  Or if you prefer a separate PR, I can send a one for that.

Reply:
I simply added this here to make the Python 3 environment in the container match the Python 2 environment, which I though would help you in your efforts to convert restore_neighbors.py to Python 3. I'd prefer if you create a separate PR to handle docker-sonic-vs.

Reply:
In this case, I would think it is better to handle python3 dependencies for restore_neighbors.py in this file and docker-sonic-vs file together.  I,E, you can drop these changes here, and I will open a PR for both. That way, it is easier to track the PR dependencies while merging and debug etc.

Reply:
Agreed. Removed.